src/NServiceBus.AcceptanceTests/PipelineExt/MutingHandlerExceptions.cs

```diff
@@ -52,7 +51,7 @@ class MyExceptionFilteringBehavior : Behavior<PhysicalMessageProcessingContext>
                         //invoke the handler/rest of the pipeline
                         await next().ConfigureAwait(false);
                     }
-                    //catch specifix exceptions or
+                        //catch specifix exceptions or
```

Comment:
@SimonCropp Seems like R# autoformat did this, do you know if it can be changed? (Can't seem to find the setting)